datascience/exchange.ipynb

```diff
@@ -0,0 +1,319 @@
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<div>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>net_flow</th>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>country_to</th>\n",
+       "      <th></th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>BE</th>\n",
+       "      <td>71154</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>GB</th>\n",
+       "      <td>31699</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>CH</th>\n",
+       "      <td>5445</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>IT</th>\n",
+       "      <td>-5891</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>DE</th>\n",
+       "      <td>-21468</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>ES</th>\n",
+       "      <td>-100527</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "            net_flow\n",
+       "country_to          \n",
+       "BE             71154\n",
+       "GB             31699\n",
+       "CH              5445\n",
+       "IT             -5891\n",
+       "DE            -21468\n",
+       "ES           -100527"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "df_exchange.groupby('country_to').sum().sort_values('net_flow',ascending=False)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "How does this evolve in time ? In the graph below, we show how electricity exchanges have evolved from November to January. We notice for instance that import and export vary with a given country. \n",
+    "\n",
+    "For instance, France exported a lot of electricity to Germany during November, but started importing more in December : "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "df_exchange['timestamp'] = df_exchange['timestamp'].apply(lambda x: arrow.get(x).datetime)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/usr/local/lib/python2.7/site-packages/ggplot/ggplot.py:192: RuntimeWarning: Setting \"mpl.rcParams['axes.prop_cycle']=cycler(u'color', [u'#333333', u'#348ABD', u'#7A68A6', u'#A60628', u'#467821', u'#CF4457', u'#188487', u'#E24A33'])\" raised an Exception: u'axes.prop_cycle is not a valid rc parameter.See rcParams.keys() for a list of valid parameters.'\n",
```

Comment:
@corradio - I would remove the output of that cell to make sure the warning does not appear on Github